.node_version

```diff
@@ -0,0 +1 @@
+v16.20.2
```

Comment:
Why 16?

Reply:
It's [still 16](https://github.com/vercel/vercel/blob/main/.github/workflows/test.yml#L12), but [some next tests run in 18](https://github.com/vercel/vercel/blob/main/utils/chunk-tests.js#L17).

While we're here, can we set [`.nvmrc`](https://github.com/nvm-sh/nvm#calling-nvm-use-automatically-in-a-directory-with-a-nvmrc-file) as well? That's what I use and [`vercel/api` already has one](https://github.com/vercel/api/blob/main/.nvmrc).